feature/staticroutes/feature.textproto

```diff
@@ -0,0 +1,28 @@
+id {
```

Comment:
hmm, you moved the feature profile to a new one but the Confiog Library above is still associated with network-instance. You need to create a separate library for static roiutes.

Reply:
Done!